### DISCUSSION TITLE
Please Help！my node vesting filcoin dosnot release to my available

### DISCUSSION THREAD

**Discussion by haheihou5** [filecoin-project/lotus#9048] [General Q&A]
befor the fvm patch,my node has some error,i terminated alot of cc sectors(about  200+ ) and some datacap 32g (maybe 1-3 sectors)
and then jul 6, the miner lost 100Tib power, 48 h after i fix the windowpost and made power up, and,no error on the miner,
but ,the vesting blance number is stuck, every day the windowpost over ,the available number still no change?
I  dont know what happen and how to fix it , the lotus-miner no more bad sectors,and log no more error msg send me,
my node is f01390330  
how can i do?

**Comment by haheihou5** [comment]
i serch  a same node
https://github.com/filecoin-project/lotus/discussions/6032
i can't understand what it meaning
or how to use that command?
![1](https://user-images.githubusercontent.com/54048258/179346851-8cb1cdd1-6340-4d80-ace5-fa2f4e4dbe1c.png)
![2](https://user-images.githubusercontent.com/54048258/179346852-8bfb64c7-8292-4340-9078-9d5677ad4292.png)

**Comment by Q-Action** [comment]
近期的线性释放已经被扣光了，所以没有释放，释放会优先扣除未来的线性释放计划的币
